src/olympia/addons/templates/addons/persona_preview.html

```diff
@@ -25,7 +25,7 @@
             {# L10n: {0} is the number of users. #}
             {{ ngettext("<strong>{0}</strong> user",
                         "<strong>{0}</strong> users",
-                        addon.persona.popularity)|f(addon.persona.popularity|numberfmt)|safe }}
+                        addon.persona.popularity)|fe(addon.persona.popularity|numberfmt)|safe }}
```

Comment:
Actually, you might be right about that. `|safe` is applied to the output of `|numberfmt` which is applied to the output of `|fe` which is applied to the text. So the whole statement is made `|safe` and we shouldn't need it here.